### PR TITLE
Added reference to Frappé (for OS X Android)

### PR DESCRIPTION
**Pull request by niftylettuce**
It's a tool for OS X that easily lets you reload developer tools menu with menubar/tray and hotkey.

It'd be great if someone forked my project and added iOS support (not sure if this is possible).

### CODE REVIEW

File: docs/Debugging.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ next: testing
 To access the in-app developer menu:
 
 1. On iOS shake the device or press `control + ⌘ + z` in the simulator.
-2. On Android shake the device or press hardware menu button (available on older devices and in most of the emulators, e.g. in [genymotion](https://www.genymotion.com) you can press `⌘ + m` to simulate hardware menu button click)
+2. On Android shake the device or press hardware menu button (available on older devices and in most of the emulators, e.g. in [genymotion](https://www.genymotion.com) you can press `⌘ + m` to simulate hardware menu button click).  You can also install [Frappé](http://getfrappe.com), a tool for OS X to shake devices remotely and binds CMD+SHIFT+R hotkey.
```
Comment (mkonicek) [inline docs/Debugging.md]: "which binds to CMD+SHIFT+R"?

I don't know how to parse the sentence: "a tool for OS X to shake devices remotely and binds CMD+SHIFT+R hotkey".

File: docs/DevelopmentSetupAndroid.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ permalink: docs/android-setup.html
 next: linux-windows-support
 ---
 
-This guide describes basic steps of the Android development environment setup that are required to run React Native android apps on an android emulator. We don't discuss developer tool configuration such as IDEs here.  
+This guide describes basic steps of the Android development environment setup that are required to run React Native android apps on an android emulator. We don't discuss developer tool configuration such as IDEs here.
```
Comment (mkonicek) [inline docs/DevelopmentSetupAndroid.md]: Thanks!

File: docs/Debugging.md (hunk 1 above)
Comment (satya164) [inline docs/Debugging.md]:
```
You can also install [Frappé](http://getfrappe.com), a tool for OS X, which allows you to emulate shaking of devices remotely. You can use `⌘ + Shift + R` as a shortcut to trigger a **shake** from Frappé.
```

**Comment by niftylettuce** [comment]
@satya164 Fixed per your recommendation

**Comment by skevy** [comment]
Looks good. Thanks @niftylettuce!

**Pull request merged**